wolfcrypt/src/port/arm/armv8-32-aes-asm_c.c

```diff
@@ -44,11 +44,15 @@
 #ifdef __IAR_SYSTEMS_ICC__
 #define __asm__        asm
 #define __volatile__   volatile
+#define WOLFSSL_NO_VAR_ASSIGN_REG
 #endif /* __IAR_SYSTEMS_ICC__ */
 #ifdef __KEIL__
 #define __asm__        __asm
 #define __volatile__   volatile
 #endif /* __KEIL__ */
+#ifdef __ghs__
+#define WOLFSSL_NO_VAR_ASSIGN_REG
```

Comment:
We need these __ghs__ defines to get to the next error below:
```
#define __asm__        __asm
#define __volatile__
```

```
wolfcrypt/src/port/arm/armv8-32-aes-asm_c.c", line 426: error #18:
          expected a ")"
          : [ks] "+r" (ks), [rounds] "+r" (rounds),
          ^
```

Reply:
fixed __asm and __volatile__